### PR TITLE
Allow custom <video> aspect ratio

### PR DESCRIPTION
**Pull request by jakearchibald**
This is part of https://github.com/GoogleChrome/webdev-infra/issues/28, to allow custom aspect ratio on `<video>` and reduce CLS.

### CODE REVIEW

**Comment by jakearchibald** [comment]
I'm not sure how to interpret the lighthouse failure here.

File: site/_scss/_global.scss [hunk 1]
```diff
@@ -23,5 +23,9 @@ main:focus {
 }
 
 video {
+  // These custom properties are set by the Video shortcode if both the width
+  // and height attributes are set.
+  aspect-ratio: var(--vid-width) / var(--vid-height);
```
Comment (bramus) [inline site/_scss/_global.scss:28]: What if these attributes are not set? Should be add fallback values or simply allow this value to become invalid?

Comment (jakearchibald) [reply site/_scss/_global.scss:28]: The fallback would be the default object size of 300x150 right?

Comment (jakearchibald) [reply site/_scss/_global.scss:28]: Falling back to invalid, which is a no-op, is probably more compatible with existing content on the site.

**Comment by jakearchibald** [comment]
I haven't tested this btw, so it'll need an eyeball to check that it works with existing video content.

**Pull request closed without merging**